app/controllers/application_controller.rb

```diff
@@ -57,8 +57,8 @@ def decorated_user
     @_decorated_user ||= current_user.decorate
   end
 
-  def after_sign_in_path_for(user)
-    stored_location_for(user) || profile_path
+  def after_sign_in_path_for(_current_user = nil)
```

Comment:
as a naming nit, it seems weird to me to have this method end with `_for` if the last argument is optional? The code looks like it's `for` nothing? WDYT of removing the `_for`?